.stylelintrc

```diff
@@ -18,6 +18,8 @@
         "selector-attribute-operator-space-before": "always",
         "selector-attribute-operator-space-after": "always",
         "selector-attribute-brackets-space-inside": "never",
+        "selector-class-pattern": null,
+        "selector-id-pattern": null,
```

Comment:
Some of classnames/ids we're using don't pass the "kebab-case", which is the standard config now:
`.SearchIssue-filter`
`.popup-modal.is--visible`
`.popup-modal__close`
`#js-ReportForm`
`.js-Issue-comment-body`
`.label-editorLauncher`
...


```
webcompat/static/css/src/filter.css
 1:6  ✖  Expected class selector to be kebab-case  selector-class-pattern

webcompat/static/css/src/issue-wizard-popup.css
 58:13  ✖  Expected class selector to be kebab-case  selector-class-pattern
 63:1   ✖  Expected class selector to be kebab-case  selector-class-pattern

webcompat/static/css/src/issue-wizard.css
  23:1  ✖  Expected id selector to be kebab-case  selector-id-pattern
  28:1  ✖  Expected id selector to be kebab-case  selector-id-pattern
 833:3  ✖  Expected id selector to be kebab-case  selector-id-pattern
 906:3  ✖  Expected id selector to be kebab-case  selector-id-pattern
 911:3  ✖  Expected id selector to be kebab-case  selector-id-pattern

webcompat/static/css/src/issue.css
 106:1  ✖  Expected class selector to be kebab-case  selector-class-pattern
 155:1  ✖  Expected class selector to be kebab-case  selector-class-pattern
 159:1  ✖  Expected class selector to be kebab-case  selector-class-pattern

webcompat/static/css/src/label-editor.css
 116:1   ✖  Expected class selector to be kebab-case  selector-class-pattern
 127:1   ✖  Expected class selector to be kebab-case  selector-class-pattern
 128:1   ✖  Expected class selector to be kebab-case  selector-class-pattern
 149:1   ✖  Expected class selector to be kebab-case  selector-class-pattern
 149:33  ✖  Expected class selector to be kebab-case  selector-class-pattern


16 problems (16 errors, 0 warnings)
```

I've disabled it for now, but it probably should be addressed to have a unified naming convention.